gazebo/rendering/Camera.hh

```diff
@@ -206,6 +206,14 @@ namespace gazebo
               const double _intrinsicsS,
               const double _clipNear, const double _clipFar);
 
+      /// \brief Compute the intrinsic camera matrix, this matrix is different
+      ///        than the one used by OpenGL internally and contains the camera
+      ///        calibrated values
+      /// \return intrinsic matrix
+      private: ignition::math::Matrix3d BuildIntrinsicMatrix(
```

Comment:
Seems odd to have a private method in the middle of the public methods. Consider moving this declaration to later in the file.